e2e/testvalues/values.go

```diff
@@ -34,3 +36,11 @@ func DefaultFee(denom string) feetypes.Fee {
 func DefaultTransferAmount(denom string) sdk.Coin {
 	return sdk.Coin{Denom: denom, Amount: sdk.NewInt(IBCTransferAmount)}
 }
+
+func TendermintClientID(id int) string {
+	return fmt.Sprint("07-tendermint-" + strconv.Itoa(id))
```

Comment:
[super nit] fmt.Sprintf handles formatting without doing it ourselves.


`return fmt.Sprintf("07-tendermint-%d", id)`